lib/components/PolicyMap/PolicyMap.css

```diff
@@ -0,0 +1,82 @@
+.container {
+  width: 100%;
+  max-width: 935px;
+  height: 539px;
+  cursor: pointer;
+  border: 1px gray solid;
```

Comment:
```suggestion
  border: 1px var(--vds-color-slate-darker) solid;
  border-radius: 5px;
```

Reply:
ah good catch

Reply:
@NathanAB i see u changed the border color, just a reminder to add the border-radius line too if u haven't already

Reply:
oh oops, missed that. done